components/Header/Header.tsx

```diff
@@ -117,6 +120,13 @@ export function Header() {
                     <Link href="/wallet-settings">delegate wallet</Link>
                   </>
                 )}
+                {isDelegator && (
+                  <>
+                    <Dot />
+                    &nbsp;&nbsp;Connected as{" "}
```

Comment:
Using spaces like this is an antipattern that can make things get a bit weird.

Rather remove the `&nbsp;` and add margin to the `<Dot />` like so:

```
const Dot = styled.span`
  height: 8px;
  width: 8px;
  // change
  margin-right: 8px;
  // other suggestion
  background-color: var(--color);
  border-radius: 50%;
  display: inline-block;
`;
```

Note: I also suggested changing the background color to `var(--color)` so that it can inherit the color of the notification and therefore also be used in red notifications too with no other modifications.

Reply:
alright ill give it a try